src/test-utils/dom/collection-preferences/sticky-columns-preference.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+import FormFieldWrapper from '../form-field';
+import RadioGroupWrapper from '../radio-group';
+import RadioButtonWrapper from '../radio-group/radio-button';
+import styles from '../../../collection-preferences/styles.selectors.js';
+
+export default class StickyColumnsPreferenceWrapper extends ComponentWrapper {
```

Comment:
Can this be simplified by introducing some StickyColumnsPreferenceSideWrapper?

E.g.
```
findCollectionPreferences().findStickyColumnsPreferenceLeft().findTitle()
findCollectionPreferences().findStickyColumnsPreferenceRight().findTitle()
```